pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -61,7 +70,7 @@ stdenv.mkDerivation rec {
   base = callPackage ./base.nix {
     inherit versions stable baseName;
     inherit wxGTK python wxPython;
-    inherit debug withI18n withOCCT oceSupport ngspiceSupport scriptingSupport;
+    inherit debug withI18n withOCCT withOCE withNgspice withScripting;
```

Comment:
```suggestion
    inherit debug withI18n withOCC withOCE withNgspice withScripting;
```

Reply:
Or maybe use `withOCCT` on `base.nix` as well.

Reply:
Ah right, forgot about default.nix. Isn't OCCT confusing here though since the kicad cmake parameter is OCC?

Reply:
Or, better to keep the interface the same, hence using OCCT?

Reply:
Yea... It's up to you whether to change this API as well. I would. With a similar deprecation warning as the others.